SenderEmail/src/main/java/com/example/SenderEmail/SenderEmailApplication.java

```diff
@@ -1,37 +1,24 @@
 package com.example.SenderEmail;
 
-import com.example.SenderEmail.model.Email;
-import com.example.SenderEmail.service.EmailService;
+import io.github.cdimascio.dotenv.Dotenv;
+import io.github.cdimascio.dotenv.DotenvEntry;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.ApplicationContext;
-
-import java.util.ArrayList;
 
 @SpringBootApplication
 public class SenderEmailApplication {
-    private EmailService emailService;
-
-    public SenderEmailApplication(EmailService emailService) {
-        this.emailService = emailService;
-    }
 
     public static void main(String[] args) {
-        // SpringApplication.run(SenderEmailApplication.class, args);
 
-        ApplicationContext context = SpringApplication.run(SenderEmailApplication.class, args);
-        SenderEmailApplication app = context.getBean(SenderEmailApplication.class);
-//        app.Run();
+        // Add Environment Variables to the application context before start
+        Dotenv dotenv = Dotenv.load();
+        for (DotenvEntry entry : dotenv.entries()) {
+            System.setProperty(entry.getKey(), entry.getValue());
+        }
+
+        // Start the application
+        SpringApplication.run(SenderEmailApplication.class, args);
```

Comment:
No need of this. Why did you add  it?

You can get it working with the default main class file generated by IntelliJ as in [otp-service](https://github.com/MohanadKh03/otp-service/blob/master/src/main/java/com/ittovative/otpservice/OtpServiceApplication.java) demo.

Reply:
this use to initialize  ** dotenv ** , i will try to implement in another file

Reply:
> this use to initialize  and load ** dotenv ** , i will try to implement in another file